redis.conf

```diff
@@ -79,27 +79,20 @@
 # running on).
 #
 # IF YOU ARE SURE YOU WANT YOUR INSTANCE TO LISTEN TO ALL THE INTERFACES
-# JUST COMMENT OUT THE FOLLOWING LINE.
+# COMMENT OUT THE FOLLOWING LINE AND SET protected-mode TO no.
```

Comment:
maybe you want to refer to the `CONFIG SET` command?
i think the previous test was enough.. but if you wanna be more explicit..

Reply:
Rephrased.

Reply:
i'm having second thoughts about this addition. it seems to encourage people to turn off protected-mode, which some may do without reading the details about what it is.
maybe the original text is better (just tells people to change bind, without mentioning protected mode)?
1. if they already have a password, they're already good, no need to turn off protected mode.
2. if they don't have a password, maybe they should set one rather than mess with protected mode?

actually since the recent changes there's protected-mode and bind are not longer related to each other, so why mention it at all?

Reply:
ohh, i just realize that you're suggesting to turn it `on`, but it is on by default.. 
so is it because you're concerned that someone disabled it while bind was only on local interfaces someone disabled it, so now when going to a more exposed bind, he needs to revert it and sent p-mode back to on?
but if that's the case, why would someone turn p-mode off when bind is set only to local interfaces?

Reply:
@oranagra It said `no`, not `on`. But anyway I've rephrased it now not to propose an insecure setting as a first option.